client/admin.go

```diff
@@ -180,21 +168,25 @@ func (c *GrpcAdminClient) ListPolicies(ctx context.Context, opts ...ListOpt) ([]
 		return nil, fmt.Errorf("could not list policies: %w", err)
 	}
 
-	if len(p.PolicyIds) == 0 {
-		return nil, nil
-	}
+	return p.PolicyIds, nil
+}
 
-	getReq := &requestv1.GetPolicyRequest{
-		Id: p.PolicyIds,
+func (c *GrpcAdminClient) GetPolicy(ctx context.Context, ids ...string) ([]*policyv1.Policy, error) {
+	req := &requestv1.GetPolicyRequest{
+		Id: ids,
 	}
-	if err := getReq.Validate(); err != nil {
-		return nil, fmt.Errorf("could not validate get policy request: %w", err)
+	if err := req.Validate(); err != nil {
+		valErr := new(requestv1.GetPolicyRequestValidationError)
+		ok := errors.As(err, &valErr)
+		if !ok || valErr.Reason() != errMsgItemsLen {
+			return nil, fmt.Errorf("could not validate get policy request: %w", err)
+		}
```

Comment:
Why check for this specific error? If `Validate` returns an error, that means the request is invalid and we just need to return that error back to the user.